performance/shopify/shop_filter.rb

```diff
@@ -14,11 +14,15 @@ def shopify_asset_url(input)
   end
 
   def script_tag(url)
-    %(<script src="#{url}" type="text/javascript"></script>)
+    %(<script src="#{url}"></script>)
   end
 
   def stylesheet_tag(url, media = "all")
-    %(<link href="#{url}" rel="stylesheet" type="text/css"  media="#{media}"  />)
+    html = []
+    html << %(<link href="#{url}" rel="stylesheet")
+    html << %(media="#{media}") if media != "all"
+    html << "/>"
+    html.join(" ")
   end
```

Comment:
Not sure this is the right approach, as this isn't respecting the user's input when they explicitly specify `media = "all"`.

Another approach would be:

```suggestion
  def stylesheet_tag(url, media = nil)
    html = []
    html << %(<link href="#{url}" rel="stylesheet")
    html << %(media="#{media}") if media
    html << "/>"
    html.join(" ")
  end
```